aapcs64/aapcs64.rst

```diff
@@ -1468,10 +1613,74 @@ Table 7: Short vector extended types
 | __Bfloat16x8\_t | bfloat16x8\_t     | half-precison Brain float| 8         |
 +-----------------+-------------------+--------------------------+-----------+
 
-C++ Mangling
-------------
+APPENDIX Support for Scalable Vectors
+=====================================
+
+As an optional extension, the AArch64 architecture supports a number of
+scalable-vector operations. To facilitate accessing these operations
+from C and C++ a number of extended types need to be added to the
```

Comment:
I'd add a comma.
```suggestion
from C and C++, a number of extended types need to be added to the
```

Reply:
OK, will change.  Note that this is deliberately apeing the style for the Advanced SIMD stuff.